pkg/test/resourcefixture/testdata/basic/compute/v1beta1/computeimage/_generated_object_computeimage.golden.yaml

```diff
@@ -1,17 +1,3 @@
-# Copyright 2024 Google LLC
```

Comment:
nit: seems it's removed during re-generation, can we put it back?

Reply:
This is actually intended to be removed: https://github.com/GoogleCloudPlatform/k8s-config-connector/commit/79bcfe20d370d7eb8bde8c838b4b5300c4a21928